paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/SortCompactAction.java

```diff
@@ -116,16 +116,12 @@ public void build(StreamExecutionEnvironment env) {
         DataStream<RowData> source = sourceBuilder.withEnv(env).withContinuousMode(false).build();
         TableSorter sorter =
                 TableSorter.getSorter(env, source, fileStoreTable, sortStrategy, orderColumns);
-        DataStream<RowData> sorted = sorter.sort();
 
-        FlinkSinkBuilder flinkSinkBuilder = new FlinkSinkBuilder(fileStoreTable);
-        flinkSinkBuilder.withInput(sorted).withOverwritePartition(new HashMap<>());
-        String sinkParallelism = tableConfig.get(FlinkConnectorOptions.SINK_PARALLELISM.key());
-        if (sinkParallelism != null) {
-            flinkSinkBuilder.withParallelism(Integer.parseInt(sinkParallelism));
-        }
-
-        flinkSinkBuilder.build();
+        new FlinkSinkBuilder(fileStoreTable)
+                .withInput(sorter.sort())
+                // we pass an object here to tag it an overwrite action.
```

Comment:
```suggestion
                // This should use empty map to tag it on overwrite action, otherwise there is no overwrite action.
```

Reply:
@JingsongLi, this invoker makes confusing to me, could we optimize the logic of `commitMultiple`? IMO, null or empty map should be the same behavior in which there is no overwrite action.